src/style.css

```diff
@@ -0,0 +1,25 @@
+@import "./stars.css";
+@import "./card.css";
+
+/* #region Defalt text style */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  color: #060b35;
+}
+/* #endregion */
+
+/* #region Reset style */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
You can write reset together.  For example body, ul, h2 {  margin: 0; }. If lint agrees, you can write the second selector next line.